Code/EnginePlugins/RmlUiPlugin/Implementation/RmlUiRenderer.cpp

```diff
@@ -0,0 +1,180 @@
+#include <RmlUiPluginPCH.h>
+
+#include <RendererCore/Pipeline/RenderDataBatch.h>
+#include <RendererCore/Pipeline/ViewData.h>
+#include <RendererCore/RenderContext/RenderContext.h>
+#include <RendererCore/Shader/ShaderResource.h>
+#include <RmlUiPlugin/Implementation/RmlUiRenderData.h>
+#include <RmlUiPlugin/Implementation/RmlUiRenderer.h>
+
+#include <RendererCore/../../../Data/Plugins/Shaders/RmlUiConstants.h>
```

Comment:
You should put an Rml folder into the plugins folder, and then put shaders there.

Reply:
I also need a shader permutation and they need to be in Shaders/PermutationVars, so I just put the shaders in the same folder.

Reply:
As discussed offline leaving it like this for now